core/conf/conf_test.go

```diff
@@ -0,0 +1,54 @@
+package conf
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func setUpConfig(name, content string) {
+
+	ioutil.WriteFile(filepath.Join(getConfigDir(), name), []byte(content), 0744)
```

Comment:
It's bad practice to involve storage operations in unit-tests. If you want to test config reader, just split part with reading and part with parsing config